go.mod

```diff
@@ -1,8 +1,6 @@
 module github.com/anchore/grype
 
-go 1.21
-
-toolchain go1.21.0
+go 1.21.0
```

Comment:
This is not really the change I expected on an "update syft" PR. Anyone have context here? Is this PR just old?

Reply:
It's automated, it only looks at the latest release version of syft - but grype is currently using an unreleased version so it just doesn't know how to handle it.  It'll create a new PR every day until there is a release of syft that is actaully greater than the current commit

Reply:
Oh nevermind, I guess there wasn't a bump of syft in grype yet so I don't really know - but I think you should just ignore this one and see if it goes away eventually

Reply:
Or there may be some additional change needed to https://github.com/anchore/grype/blob/main/.github/workflows/update-syft-release.yml after the go 1.21 bump